Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -22,24 +23,24 @@ void Start()
     {
         rb = GetComponent<Rigidbody>();
         rends = GetComponents<Renderer> ();
+        plateShader = Shader.Find("PlateShake");
     }
 
     void Update()
     {
         distance = Vector3.Distance(transform.position, GameManager.Instance.GetPlayerTransform().position);
+
         if (distance <= dropRadius)
         {
-            rb.AddForce(direction * thrust);
-            rb.useGravity = true;
+            rb.AddForce(thrustDirection * thrustForce);
             NotifyChasingNurse();
         } 
-        else if (distance <= (dropRadius + 3))
-        {
-            Shake(maxShake);
-        } 
         else if (distance <= shakeRadius)
         {
-            float dynamicShake = Mathf.Lerp(maxShake, minShake, Mathf.Clamp(distance/5.0f, 0f, 1f));
+            float clampedDistance = Mathf.Clamp(distance, dropRadius, shakeRadius);
+            float lerpedDistance = Mathf.Lerp(0f, 1f, (clampedDistance - dropRadius) / (shakeRadius - dropRadius));
+            float dynamicShake = Mathf.Lerp(maxShake, minShake, lerpedDistance);
+            print(dynamicShake);
             Shake(dynamicShake);
         } 
         else 

```

Comment:
I think the else if above also covers this case? Can you try removing this and see if it causes any issues?
Then you can just make the above statement the else statement.

Reply:
yes!! sorry i didn't take out the print also my baddd

Reply:
It works! Although it starts at 0.1? It's not a big deal because it's minimal flashing for the user.. but maybe for the hoarding room and stuff we may have to tweak if there are lots of falling/loud objects!